inc/relation.constant.php

```diff
@@ -665,6 +665,7 @@
         'glpi_users'                       => 'entities_id',
         'glpi_vlans'                       => 'entities_id',
         'glpi_wifinetworks'                => 'entities_id',
+        'glpi_itilvalidationtemplates'     => 'entities_id',
```

Comment:
We try to order entries by alphabetically order, please move these new entries according to this.